entity/match/match_entity.go

```diff
@@ -0,0 +1,5 @@
+package match_entity
+
+type Match struct {
+	Id, Message, CatIssuerId, CatReceiverId, CreatedAt string
+}
```

Comment:
`status`-nya ga dimasukin ke struct sekalian mas?

Reply:
sebenernya blm dibutuhin si tp gapapa gw tambain nanti just in case kedepannya butuh

Reply:
oke mas, dari aku itu aja sih.